teku/src/main/java/tech/pegasys/teku/cli/options/DepositOptions.java

```diff
@@ -60,10 +60,9 @@ public class DepositOptions {
   private String depositSnapshotPath;
 
   @Option(
-      names = {"--Xdeposit-snapshot-enabled"},
+      names = {"--deposit-snapshot-enabled"},
```

Comment:
we need a better story around how we do these options i think.

I won't want this to be a breaking change for everyone, potentially we need a strategy...  Maybe what we do is accept the -X still, have it print a deprecation warning and then do the same functionality. This would mean that we have 2 flags until we remove the -X flag in a couple of releases.. .thoughts?

I guess my main concern is i'd prefer to have extra work than break user configuration...

Reply:
If we do it this way though, it'll need to be listed as a breaking change... the alternative i suggested could be an 'upcoming breaking change'

Reply:
@rolfyone I'm on your side :joy:
I will try to do it in this way

Reply:
This approach sounds reasonable to me. And the --X option won't show up in the standard help option.

Reply:
> I've not find any way to make it work on one PicoCli `Option` item

I was thinking 2 options to have the separation... and then we can just delete the -X in a future release